src/main/java/com/laserfiche/repository/api/RepositoryApiClientImpl.java

```diff
@@ -51,6 +50,15 @@ private <C> C createClient(Class<C> clientInterface) {
                 .create(clientInterface);
     }
 
+    public void setDefaultRequestHeaders(Headers defaultHeaders) {
```

Comment:
with this implementation, is it possible to overwrite/remove the default headers if this is called multiple times? Maybe we need to have Headers as a property in this class